chapters/memory-layout/c-assembly-interaction/reading/README.md

```diff
@@ -0,0 +1,48 @@
+# C - Assembly Interaction
+
+Considering that assembly language poses challenges both in reading and in developing code, the general trend is to migrate towards high-level languages (which are much easier to read and provide a more user-friendly API).
+However, there are still situations where, for optimization reasons, small assembly routines are used and integrated into the high-level language module.
+
+In this laboratory, we will explore how assembly modules can be integrated into C programs and vice versa.
+
+## Using Assembly Procedures in C Functions
+
+Using assembly procedures in C functions for a C program to be executed, it must be translated into the machine code of the processor; this is the task of a compiler.
+Since this compiled code is not always optimal, in some cases it is preferable to replace portions of code written in C with portions of assembly code that do the same thing, but with better performance.
+
+### Declaration of the Procedure
+
+In order to ensure that the assembly procedure and Module C are properly combined and compatible, the following steps must be followed:
+    -declare the procedure label as global, using the GLOBAL directive. In addition to this, any data that will be used by the procedure must be declared as global.
```

Comment:
```suggestion
* declare the procedure label as global, using the GLOBAL directive. In addition to this, any data that will be used by the procedure must be declared as global.
```

Reply:
Use `-` not `*` for unnumbered lists. But otherwise, yes, align lists to the left, without tabs.